CHANGELOG.md

```diff
@@ -1,4 +1,19 @@
-# 2.0.0 (2018-01-04)
+# 3.0.0 (2018-03-20)

```

Comment:
Should date be 2018-03-19?

Reply:
```
git log

commit 86b65776aaa3173fd15a98016a00ef18c1132f81
Merge: f99d364 d9775a5
Author: Michael Carroll <juwara0@users.noreply.github.com>
Date:   Tue Mar 20 07:38:43 2018 -0700

    Merge pull request #22 from juwara0/updateDependencies
    
    Update dependencies
```

Reply:
Oh, sorry, the PR was _opened_ two days ago - my mistake

https://github.com/ciena-frost/ember-frost-action-bar/pull/22#issue-176032016